<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSS</class>
 <widget class="QWidget" name="RSS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>447</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="newFeedButton">
       <property name="text">
        <string>New subscription</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/Icons/oxygen/subscribe.png</normaloff>:/Icons/oxygen/subscribe.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="markReadButton">
       <property name="text">
        <string>Mark items read</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/Icons/oxygen/button_ok.png</normaloff>:/Icons/oxygen/button_ok.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="updateAllButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Refresh RSS streams</string>
       </property>
       <property name="text">
        <string>Update all</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/Icons/oxygen/view-refresh.png</normaloff>:/Icons/oxygen/view-refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_h">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="news_lbl">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Torrents:&lt;/span&gt; &lt;span style=&quot; font-style:italic;&quot;&gt;(double-click to download)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_v">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTreeWidget" name="listNews">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">Bullet</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Article title</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Feed URL</string>
           </property>
          </column>
         </widget>
         <widget class="QTextBrowser" name="textBrowser"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/oxygen/remove.png</normaloff>:/Icons/oxygen/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/oxygen/edit_clear.png</normaloff>:/Icons/oxygen/edit_clear.png</iconset>
   </property>
   <property name="text">
    <string>Rename...</string>
   </property>
   <property name="toolTip">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/oxygen/view-refresh.png</normaloff>:/Icons/oxygen/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
  </action>
  <action name="actionNew_subscription">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/oxygen/subscribe16.png</normaloff>:/Icons/oxygen/subscribe16.png</iconset>
   </property>
   <property name="text">
    <string>New subscription...</string>
   </property>
  </action>
  <action name="actionUpdate_all_feeds">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/oxygen/view-refresh.png</normaloff>:/Icons/oxygen/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Update all feeds</string>
   </property>
   <property name="toolTip">
    <string>Update all feeds</string>
   </property>
  </action>
  <action name="actionMark_items_read">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/oxygen/button_ok.png</normaloff>:/Icons/oxygen/button_ok.png</iconset>
   </property>
   <property name="text">
    <string>Mark items read</string>
   </property>
   <property name="toolTip">
    <string>Mark items read</string>
   </property>
  </action>
  <action name="actionDownload_torrent">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/oxygen/download.png</normaloff>:/Icons/oxygen/download.png</iconset>
   </property>
   <property name="text">
    <string>Download torrent</string>
   </property>
  </action>
  <action name="actionOpen_news_URL">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/url.png</normaloff>:/Icons/url.png</iconset>
   </property>
   <property name="text">
    <string>Open news URL</string>
   </property>
  </action>
  <action name="actionCopy_feed_URL">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/oxygen/edit-copy.png</normaloff>:/Icons/oxygen/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy feed URL</string>
   </property>
  </action>
  <action name="actionRSS_feed_downloader">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/oxygen/download.png</normaloff>:/Icons/oxygen/download.png</iconset>
   </property>
   <property name="text">
    <string>RSS feed downloader...</string>
   </property>
  </action>
  <action name="actionNew_folder">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/oxygen/folder-new.png</normaloff>:/Icons/oxygen/folder-new.png</iconset>
   </property>
   <property name="text">
    <string>New folder...</string>
   </property>
  </action>
  <action name="actionManage_cookies">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/Icons/oxygen/cookies.png</normaloff>:/Icons/oxygen/cookies.png</iconset>
   </property>
   <property name="text">
    <string>Manage cookies...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
